x/fungible/keeper/system_contract.go

```diff
@@ -87,6 +87,10 @@ func (k *Keeper) GetWZetaContractAddress(ctx sdk.Context) (ethcommon.Address, er
 	if err := sysABI.UnpackIntoInterface(&wzetaResponse, "wZetaContractAddress", res.Ret); err != nil {
 		return ethcommon.Address{}, cosmoserrors.Wrapf(types.ErrABIUnpack, "failed to unpack wZetaContractAddress: %s", err.Error())
 	}
+
+	if wzetaResponse.Value == ethcommon.HexToAddress("0x0") {
```

Comment:
There are many places using `if some_address == ethcommon.Address{}`, should we use a unified pattern?